codegenerator/cli/templates/static/codegen/src/db/TablesStatic.res

```diff
@@ -119,7 +119,7 @@ module RawEvents = {
     @as("block_timestamp") blockTimestamp: int,
     @as("block_fields") blockFields: Js.Json.t,
     @as("transaction_fields") transactionFields: Js.Json.t,
-    params: Js.Json.t,
+    params: string,
```

Comment:
It failed with the `PostgresError: column "params" is of type json but expression is of type boolean`

Reply:
Ah interesting. Shouldn't it always be in a json object though and not a value?